libraries/AP_HAL_ChibiOS/hwdef/AnyleafH7/README.md

```diff
@@ -0,0 +1,75 @@
+# AnyLeaf Mercury H7 Flight Controller
+
+The Mercury H7 is a flight controller produced by [AnyLeaf](http://www.anyleaf.org/).
+
+## Features
+    Processor
+        STM32H743 32-bit processor
+    Sensors
+        ICM42688 Acc/Gyro with dedicated 32.768kHz crystal
+        DPS310 barometer
+    Power
+        2S - 6S Lipo input voltage with voltage monitoring
+        9V, 3A supply for powering video transmitters
+        5V, 2A supply for powering servos and electronics
+        3.3V, 1A supply for powering electronics
+    Interfaces
+        8x Bidirectional-DSHOT, or PWM-capable motor outputs
+        1x CAN-FD port for external peripherals
+        1x DJI-format Vtx connector
+        5x UARTs/serial for external peripherals
+        1x I2C for external peripherals
+        USB-C port
+        All UARTS support hardware inversion. SBUS, SmartPort, and other inverted protocols work on any UART without “uninvert hack”
+        Onboard ExpressLRS radio tranceiver
+
+## Pinout
+
+Pins and connector values are labeled on the flight controller PCB.
```

Comment:
should include images and link in this readme....and all pins are not labeled on the PCB..where is TX2 and RX2 (DJI connector)...same for ESC connector telem pin (RX3)..

Reply:
I added some clarification text; LMK how you'd like to proceed. I can draft up some images if you'd like, but I think now after the updates, they are probably not required in conjunction with the readme, datasheet, and silkscreen text. Your call.

Reply:
when I get a chance in the next few days...I will pull this down when you think you've done everything you can and make some changes and push back up

Reply:
just pushed readme changes....outputs need to be addressed by @andyp1per and other changes made to defaults